src/StackExchange.Redis/KeyspaceIsolation/DatabaseWrapper.cs

```diff
@@ -10,571 +10,349 @@ public DatabaseWrapper(IDatabase inner, byte[] prefix) : base(inner, prefix)
         {
         }
 
-        public IBatch CreateBatch(object asyncState = null)
-        {
-            return new BatchWrapper(Inner.CreateBatch(asyncState), Prefix);
-        }
+        public IBatch CreateBatch(object? asyncState = null) =>
```

Comment:
In the first passes _everything_ was touched so this was less noise, but now: yeah...just want to get that done but I can separate out as well. I also want to rename these 2 wrapper classes because vs. the Async variants they're just not right.

Reply:
Yeah, That's a judgment call, you've done the legwork already in here, might as well keep it in if it's cleanup you want to do anyway unless you can just explicitly cherry-pick this out and not have it be a ton of work.